ada_2_fhir-r4/zibs2020/ada_project_drivers/HealthcareProvider-driver.xsl

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright © Nictiz
+
+This program is free software; you can redistribute it and/or modify it under the terms of the
+GNU Lesser General Public License as published by the Free Software Foundation; either version
+2.1 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Lesser General Public License for more details.
+
+The full text of the license is available at http://www.gnu.org/copyleft/lesser.html
+-->
+
+<xsl:stylesheet exclude-result-prefixes="#all"
+    xmlns="http://hl7.org/fhir"
+    xmlns:util="urn:hl7:utilities" 
+    xmlns:f="http://hl7.org/fhir" 
+    xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl"
+    xmlns:nf="http://www.nictiz.nl/functions" 
+    xmlns:uuid="http://www.uuid.org"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema" 
+    xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
+    version="2.0">
+    
+    <xsl:import href="../payload/0.0/all_zibs.xsl"/>
+    
+    <xsl:param name="referById" as="xs:boolean" select="true()"/>
```

Comment:
Volgens mij is dit 'de oplossing' voor de variabele `$referByReferencingId` zoals voorgesteld door Pieter. In STU3 was deze parameter al global

Reply:
True, true